src/style.css

```diff
@@ -1 +1,34 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__block {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
background-shorthand could be used
https://www.w3schools.com/css/css_background_shorthand.asp

Reply:
done